sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -2839,19 +2841,27 @@ public void testAbsoluteProcessingTimeTimerRejected() throws Exception {
 
             @ProcessElement
             public void processElement(@TimerId(timerId) Timer timer) {
-              timer.set(new Instant(0));
+              try {
+                timer.set(new Instant(0));
+                fail("Should have failed due to processing time with absolute timer.");
```

Comment:
The error message is a b it confusing. Should it refer to "invalid processing time"?

Reply:
Yes, but not in processing time: https://github.com/apache/beam/commit/86bcf26c965f876be2e67aea769de3406ef25a97#diff-95b3ede99105b81444964de712a52386R967